app/views/posts/index.html.slim

```diff
@@ -6,8 +6,10 @@
                                      params[:section].titleize.capitalize => "" }
 
 .govuk-grid-row
-  h1.govuk-heading-xl = t(".#{params[:section]}.title")
-  p.govuk-body = t(".#{params[:section]}.description")
+  .govuk-grid-column-full
+    h2.govuk-heading-xl = t(".#{params[:section]}.title")
```

Comment:
:raising_hand: Having an `h2`title without `h1` is an accessibility issue (we fixed a few of those after our accessibility review)

![image](https://github.com/user-attachments/assets/72ad8d70-9dcb-4a78-a3e6-bd2a986152af)